cmd/asset-syncer/postgresql_utils.go

```diff
@@ -185,12 +233,15 @@ func (m *postgresAssetManager) filesExist(chartFilesID, digest string) bool {
 }
 
 func (m *postgresAssetManager) insertFiles(chartFilesID string, files models.ChartFiles) error {
```

Comment:
Do you know whether the `chartFilesID` arg here is just left over? I checked what gets imported, and the `files.ID` appears identical and so could be used to fill the field too.

Reply:
I probably missed that so no, there is not a reason